tools/e2e/Taskfile.yaml

```diff
@@ -41,6 +40,17 @@ tasks:
       - git checkout main
       - git pull
 
+  nvm-install:
+    dir: "{{.DATALENS_UI_FOLDER}}"
+    desc: Install nvm & node
+    cmds:
+      - |
+        PROFILE=/dev/null bash -c \
+          'curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash && \
+          export NVM_DIR="$HOME/.nvm" && \
+          source $NVM_DIR/nvm.sh && \
+          nvm install'
```

Comment:
The problem with this is that nvm doesn't properly work with subshells and you have to reactivate it everytime inside task command :( Otherwise global version is used and not the one of the repository set by `.nvm` file

Let's just probably install fixed version of node and update it from time to time, same as datalens-ui do it:
https://github.com/datalens-tech/datalens-ui/blob/9f6210a6c5db2d8003cb041c45fc6ba17d159221/.github/workflows/e2e_tests.yml#L37C1-L41C53

Reply:
If you prefer taskfile hacks to use proper npm version with nvm:
```
tasks:
  _with_nvm:
    internal: true
    cmds:
      - "source ${HOME}/.nvm/nvm.sh && nvm use && {{.COMMAND}}"
      
  test:
    ...
    cmds:
      - task: _with_nvm
        vars: { COMMAND: "npm run test:install:chromium" }
      - task: _with_nvm
        vars: { COMMAND: "npm run test:e2e:opensource" }